internal/gojs/runtime.go

```diff
@@ -114,9 +114,10 @@ func nanotime1(ctx context.Context, mod api.Module, stack []uint64) {
 //
 // See https://github.com/golang/go/blob/go1.19/src/runtime/sys_wasm.s#L188
 var Walltime = spfunc.MustCallFromSP(false, &wasm.HostFunc{
-	ExportNames: []string{functionWalltime},
-	Name:        functionWalltime,
+	ExportNames: []string{walltimeName},
+	Name:        walltimeName,
 	ResultTypes: []api.ValueType{i64, i32},
+	ResultNames: []string{"sec", "nsec"},
```

Comment:
currently gojs is the only thing that returns multiple results. However, a future change will do similar in WASI, and this is infrastructure for that.